### PR TITLE
Update users example

### PR DESCRIPTION
**Pull request by karl-johan-grahn**
For https://github.com/slack-go/slack/issues/974

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+### v0.9.6 - 26 Sept 2021
+Update users example with input examples
+
```
Comment (kanata2) [inline CHANGELOG.md]: CHANGELOG is not updated now... 😢

**Review by kanata2 — COMMENTED** [review]
Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (karl-johan-grahn) [reply CHANGELOG.md]: I'll update it with my change at least

Comment (kanata2) [reply CHANGELOG.md]: If we update it halfway, users may be confused, so let us only use the release notes for now.

Comment (kanata2) [reply CHANGELOG.md]: related: #987

**Review by kanata2 — APPROVED** [review]
Thanks.

**Pull request merged**